overview/default.nix

```
@@ -146,34 +148,79 @@ let
         '';
     };
 
-    projects = rec {
+    projects = {
       one = name: project: ''
-        <section><details><summary>${heading 2 name}</summary>
+        ${heading 1 name}
         <https://nlnet.nl/project/${name}>
 
         ${render.packages.many (pick.packages project)}
         ${render.options.many (pick.options project)}
         ${render.examples.many (pick.examples project)}
-        </details></section>
       '';
-      many = projects: concatLines (mapAttrsToList one projects);
+      # Many projects are renderes as links to their individual project sites
+      many =
+        projects:
+        concatLines (
+          mapAttrsToList (name: _: ''
+            <a href="${name}">${name}</a>
```

```suggestion
            <a href="project/${name}">${name}</a>
```

The issue discussed namespacing for better extensibility, and I agree with that.

---

Yeah agree, going for extensibility would be the safer bet here.